me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/helper/CustomTabsHelper.java

```diff
@@ -35,8 +35,8 @@ public static CustomTabsIntent getCustomTabIntent(Context context, String url) {
     if (packageName != null) {
       customTabsIntent.intent.setPackage(packageName);
     }
-    customTabsIntent.intent.setFlags(Intent.FLAG_ACTIVITY_NO_HISTORY | Intent.FLAG_ACTIVITY_CLEAR_TOP
-        | Intent.FLAG_ACTIVITY_SINGLE_TOP | Intent.FLAG_ACTIVITY_NEW_TASK);
+    customTabsIntent.intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP
+        | Intent.FLAG_ACTIVITY_SINGLE_TOP);
```

Comment:
Is it standard to be setting these flags in code? Or should they be in an XML file because they're static?